guides/source/error_reporting.md

```diff
@@ -35,17 +35,17 @@ Rails.error.handle(SomethingIsBroken) do
 end
 ```
 
-Rails wraps all executions (such as HTTP requests, jobs, and `rails runner` invocations) in the error reporter, so any unhandled errors raised in your app will automatically be reported to your error-reporting service via their subscribers.
+Rails wraps all executions (such as [HTTP requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), [jobs](https://guides.rubyonrails.org/active_job_basics.html), and [rails runner](https://guides.rubyonrails.org/command_line.html#bin-rails-runner) invocations) in the error reporter, so any unhandled errors raised in your app will automatically be reported to your error-reporting service via their subscribers.
```

Comment:
Here's some notes about how to link to [other references](https://3.basecamp.com/3076981/buckets/35498807/todos/7116145100#__recording_7182213941): 
- Link to guides should not contain the full URL but rather just the guide path. e.g. `[Active Storage Overview](active_storage_overview.html)`
- When linking on the same guide, we should be able to use the anchor to reference them e.g. `[Attribute Assignment](#attribute-assignment)`
- Keep links pointing to api instead of edgeapi, they're `[translated to edgeapi automatically when the guides are generated](https://edgeguides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#linking-to-the-api).`

Also, I'm not sure about linking to the HTTP guides, I think a link to the Mozilla documentation fine (but maybe someone else can clarify), but the [general rule](https://3.basecamp.com/3076981/buckets/35498807/todos/7116145100#__recording_7386968481) is:
- We avoid external links (where possible). If a gem is inside the rails org is fine to keep mentioning it, but if it is out of the rails org we should remove the mention. I guess the same question applies for the sentry example above, it would be useful to get clarification. 

```suggestion
Rails wraps all executions (such as [HTTP requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), [jobs](active_job_basics.html), and [rails runner](command_line.html#bin-rails-runner) invocations) in the error reporter, so any unhandled errors raised in your app will automatically be reported to your error-reporting service via their subscribers.
```